src/components/MoneyRequestHeader.tsx

```diff
@@ -151,7 +151,7 @@ function MoneyRequestHeader({report, parentReportAction, policy, shouldUseNarrow
                             success
                             medium
                             text={translate('iou.reviewDuplicates')}
-                            style={[styles.p0]}
+                            style={[styles.p0, styles.ml1]}
```

Comment:
```suggestion
                            style={[styles.p0, styles.ml2]}
```

`ml1` actually looks a little small let's bump this to `ml2`